src/cff.cc

```diff
@@ -530,6 +534,7 @@ bool ParsePrivateDictData(
         if (vsindex >= out_cff->region_index_count.size()) {
           return OTS_FAILURE();
         }
+        out_cff->vsindex_per_font.back() = vsindex;
```

Comment:
The scope of `vsindex` can be reduced now by moving the declaration to where it is set

Reply:
`vsindex` is also used when parsing `blend`, just below that block. Would you prefer changing the `blend` case to use `out_cff->vsindex_per_font.back()` in order to reduce the scope of `vsindex` as you propose? Or reduce the scope to the [`while` block](https://github.com/khaledhosny/ots/pull/239/files#diff-8a7f0692284ab2a9a55e74c3218713bbfb02005e48199f2a73aa9f15b3263b65R420) level?

Reply:
I see. Let's leave it the way it is for now.